app/components/card_component.rb

```diff
@@ -3,14 +3,27 @@ class CardComponent < ApplicationComponent
 
   private
 
-  def card_classes
+  def card_classes_content
+    [
+      "p-4",
+      "sm:p-6",
+    ].compact.join(" ")
+  end
+
+  def card_classes_wrapper
     [
       "shadow",
       "rounded-lg",
       "bg-white",
+    ].compact.join(" ")
+  end
+
+  def card_classes_footer
+    [
+      "bg-purple-50",
       "p-4",
       "sm:p-6",
-      ("rounded-b-none" if footer?)
+      ("rounded-t-none" unless content.blank?),
```

Comment:
I note that this changes the logic from checking for footer content vs checking for content overall. I trust you that there was a reason for that (I don't remember the original reason why we were checking for footer content).

Also, reminder that in a ViewComponent you can always use `content?`, instead of content.blank? (https://viewcomponent.org/api.html#content--boolean).